target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -182,7 +182,8 @@ cf-e320n-v2)
 	ucidef_set_led_wlan "wlan" "WLAN" "$board:blue:wlan" "phy0tpt"
 	;;
 cf-e380ac-v1|\
-cf-e380ac-v2)
+cf-e380ac-v2)\
+cf-e355ac)
 	ucidef_set_led_netdev "lan" "LAN" "$board:green:lan" "eth0"
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "$board:blue:wlan2g" "phy1tpt"
```

Comment:
My mistake, I wanted to write "move this line **up**"... but you could at least first check what LEDs your board has defined in mach-*.c file (in fact, you are the one who defined LEDs, so you should know what's there) and then include it in the correct place - you are here the one adding support for this board, I have never seen it.

Next time, if I write something stupid or make a typo in comment, just correct me.

As you can see:

```
cf-e320n-v2)
	ucidef_set_led_netdev "lan" "LAN" "$board:green:lan" "eth0"
	ucidef_set_led_netdev "wan" "WAN" "$board:red:wan" "eth1"
	ucidef_set_led_wlan "wlan" "WLAN" "$board:blue:wlan" "phy0tpt"
```
And your board has:

```
cf-e355ac:red:wan
cf-e355ac:green:lan
cf-e355ac:blue:wlan
```

Doesn't that look familiar for you? Of course, you should first make sure it's correct (lan/wan interfaces, radio - you have two radios on your board, maybe there are also two LEDs for both radios? The phy0tpt will be probably 5G radio, maybe you should then rename the LED).

Again, are you testing this code on a real device?